docs/batch jobs/ISEL timetable extraction batch job/isel-timetable-batch-specification.md

```diff
@@ -74,7 +76,7 @@ data class Label(
 data class Event (
 	@JsonProperty("title") val title : String,
     @JsonProperty("description") val description : String,
-    @JsonProperty("type") val type : String,
+    @JsonProperty("category") val category : String,
```

Comment:
According to the [documentation](https://github.com/i-on-project/core/blob/master/docs/api/write/insertClassSectionEvents.md), the category would be a positive integer) unique ID.